.github/workflows/wheels-test.yaml

```diff
@@ -149,17 +147,29 @@ jobs:
         sha: ${{ inputs.sha }}
 
     - name: Run tests
+      # TODO: remove 'continue-on-error' before closing https://github.com/rapidsai/build-planning/issues/3
+      continue-on-error: ${{ matrix.PY_VER == '3.11' }}
       run: ${{ inputs.script }}
       env:
         GH_TOKEN: ${{ github.token }}
 
     - name: Generate test report
+      # TODO: remove 'continue-on-error' before closing https://github.com/rapidsai/build-planning/issues/3
+      continue-on-error: ${{ matrix.PY_VER == '3.11' }}
       uses: test-summary/action@v2.2
       with:
         paths: "${{ env.RAPIDS_TESTS_DIR }}/*.xml"
         show: ${{ inputs.test_summary_show }}
       if: always()
 
     - name: Upload additional artifacts
+      # TODO: remove 'continue-on-error' before closing https://github.com/rapidsai/build-planning/issues/3
+      continue-on-error: ${{ matrix.PY_VER == '3.11' }}
       if: "!cancelled()"
       run: rapids-upload-artifacts-dir cuda${RAPIDS_CUDA_VERSION%%.*}_$(arch)_py${RAPIDS_PY_VERSION//.}
+
+    # TODO: remove before closing https://github.com/rapidsai/build-planning/issues/3
+    - name: mark Python 3.11 jobs successful
```

Comment:
I'm not 100% sure if this step is necessary, but it doesn't hurt. We have enough testing to know that this PR will help unblock CI.